webapp/channels/src/components/admin_console/billing/billing_subscriptions/cloud_trial_banner.tsx

```diff
@@ -72,40 +72,20 @@ const CloudTrialBanner = ({trialEndDate}: Props): JSX.Element | null => {
             title={(
                 <FormattedMessage
                     id='admin.subscription.cloudTrialCard.upgradeTitle'
-                    defaultMessage='Upgrade to one of our paid plans to avoid Free plan data limits'
+                    defaultMessage='Upgrade to one of our paid plans to continue using Mattermost'
```

Comment:
For all of these, the defaultMessage field is only what's shown when there is no matching translation, or the user's locale is set to one we do not have support for. You will need to also translate these changes to the matching translation in https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/i18n/en.json

The `id` field on the FormattedMessage will be 1:1 with one in the en.json file. This way, the changes also cascade out into other languages before release.